website/docs/docs/dbt-ai/setup-remote-mcp.md

```diff
@@ -10,16 +10,29 @@ id: "setup-remote-mcp"
 The remote server uses an HTTP connection and makes calls to dbt-mcp hosted on the cloud-based dbt platform. 
 
 1. Ensure that you have [AI Features](https://docs.getdbt.com/docs/cloud/enable-dbt-copilot) turned on.
-2. Obtain the following information:
+2. Obtain the following information from dbt Platform:
 
-  - **dbt Cloud host**: Use this to form the full URL. For example, replace `<host>` here: `https://<host>/api/ai/v1/mcp/`. It may look like: `https://cloud.getdbt.com/api/ai/v1/mcp/`.
+  - **dbt Cloud host**: Use this to form the full URL. For example, replace `<host>` here: `https://<host>/api/ai/v1/mcp/`. It may look like: `https://cloud.getdbt.com/api/ai/v1/mcp/`. If you have a multi-cell account, the host url will be `<ACCOUNT_PREFIX>.us1.dbt.com`. For more information, [check out our docs.](/cloud/about-cloud/access-regions-ip-addresses)
```

Comment:
```suggestion
  - **dbt Cloud host**: Use this to form the full URL. For example, replace `<host>` here: `https://<host>/api/ai/v1/mcp/`. It may look like: `https://cloud.getdbt.com/api/ai/v1/mcp/`. If you have a multi-cell account, the host URL will be in the `<ACCOUNT_PREFIX>.us1.dbt.com` format. For more information, [check out our docs.](/cloud/about-cloud/access-regions-ip-addresses)
```